2] loto/hra lotto.py

```diff
@@ -0,0 +1,41 @@
+import random
```

Comment:
cely tenhle kod se mi zda opravdu velmi komplikovany. zkus si to rozdelit do  funkci 1] to co vybere user 2] to co vybere pocitac 3] main